specification/eventgrid/resource-manager/readme.go.md

```diff
@@ -13,11 +13,19 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2018-09-preview
   - tag: package-2018-05-preview
   - tag: package-2018-01
   - tag: package-2017-09-preview
   - tag: package-2017-06-preview
 ```
+### Tag: package-2018-09-preview and go
+
+These settings apply only when `--tag=package-2018-09-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2018-09-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2018-09-15-preview/$(namespace)
```

Comment:
The closing ``` is missing causing a parsing failure.

Reply:
@jhendrixMSFT  thanks for the help. this seems to fix the parsing issue.

Reply:
@praries880 We are down to the known linter error/failure here only (https://github.com/Azure/azure-openapi-arm-validator/issues/10)